### ISSUE TITLE
Failing with oracle jdk 9

### ISSUE THREAD

**Issue by vvasuki** [swagger-api/swagger-core#2588]
Swagger serving was crashing on my server with the following java version:
```
java -version
java version "9.0.1"
Java(TM) SE Runtime Environment (build 9.0.1+11)
Java HotSpot(TM) 64-Bit Server VM (build 9.0.1+11, mixed mode)
```
The log is here: https://pastebin.com/raw/xDM2w2Gm

The fix was to add a dependency:
`"javax.xml.bind" % "jaxb-api" % "2.3.0",` as indicated by [this thread](https://stackoverflow.com/questions/43574426/how-to-resolve-java-lang-noclassdeffounderror-javax-xml-bind-jaxbexception-in-j).

@pjfanning , whose library I was using ( https://github.com/swagger-akka-http/swagger-akka-http/issues/62#issuecomment-351768423 ) , suggested that we report it here.

**Comment by frantuma** [comment]
should be fixed in https://github.com/swagger-api/swagger-core/commit/3f7d054cdfe728800cc2fd4e7aefeedc8d9a66fc

**Comment by vdua** [comment]
The commit is not merged in the 1.5 branch. Does that mean java 9/10 is not supported for 1.5.X version ?

**Comment by pjfanning** [comment]
@vdua just add the jaxb-api dependency to your own build.
